command/testdata/show-json/basic-delete/output.json

```diff
@@ -66,7 +68,9 @@
                     "id": "placeholder"
                 },
                 "after": null,
-                "after_unknown": {}
+                "after_unknown": {},
+                "after_sensitive": false,
```

Comment:
The sensitive marker is `false` here because the corresponding value is `null`. This is a result of the slightly different needs for expressing sensitive compared to unknown.

Sensitivity also results in a truncated structure as soon as it is detected. This is easiest to explain with examples (using [the `sensitive` function](27341) to indicate sensitive values).

| `before`/`after`  | `before_sensitive`/`after_sensitive` |
| ------------- | ------------- |
| `{ a = "foo" }`  | `{}`  |
| `sensitive({ a = "foo" })`  | `true`  |
| `{ a = sensitive("foo") }`  | `{ a = true }`  |
| `{ a = { b = "foo" }, c = "bar" } }` | `{}` |
| `{ a = { b = "foo" }, c = sensitive("bar") } }` | { c = true }` |

Reply:
Alright, sounds good, this is also good detail to surface as a reference - resolving this :+1: